packages/main/src/mainWindow.ts

```diff
@@ -72,6 +72,8 @@ async function createWindow() {
  * Restore an existing BrowserWindow or Create a new BrowserWindow.
  */
 export async function restoreOrCreateWindow() {
+  await app.whenReady();
```

Comment:
Sometimes, when opening the Launcher using the deep link, this method was executed before the app was ready.

This is a safeguard to avoid unexpected errors